Std/Tactic/ShowTerm.lean

```diff
@@ -30,3 +30,8 @@ elab (name := showTerm) tk:"show_term " t:term : term <= ty => do
   Term.synthesizeSyntheticMVarsNoPostponing
   addTermSuggestion tk (← instantiateMVars e).headBeta (origSpan? := ← getRef)
   pure e
+
+syntax (name := by?) "by?" tacticSeq : term
+macro_rules
+  | `(term| by? $t) => `(show_term by $t)
```

Comment:
```suggestion
  | `(term| by?%$tk $t) => `(show_term%$tk by%$tk $t)
```
double check that the message and the errors from `by` both end up on the `by?` token

Reply:
Seems to work fine (it seemed to also work ok before tbh, but I appreciate that this is probably an improvement)

Reply:
keep in mind that you have to test with `by?` on the same line as `tac`, or else you might get a misleading impression of the end of the message span

Reply:
Yeah I did check a couple of different things like that, multiline, nested by's, syntax errors later and everything seemed to behave as by did